Topics/lifecycle.md

```diff
@@ -0,0 +1,111 @@
+# The Data Science Project lifecycle
```

Comment:
Project or Process? I'm not sure either. But if we use the term Project it might make some people think something specific about Data Science - while I know what you mean, some others might not!